admin/routes/questions.js

```diff
@@ -1,10 +1,19 @@
 const express = require('express')
 const router = express.Router()
+const moment = require('moment')
 const { getQuestions } = require('../controllers/questions')
+const winstonLogger = require('./../winstonLogger')
```

Comment:
just use `const winston = require('winston')`

Reply:
Same as above, winstonLogger is a separate logger only for API calls

Reply:
there was no requirement for a separate logger.  please remove

Reply:
OK, I'll delete the separate logger and will use ```winston.info(...```